ROADMAP.md

```diff
@@ -43,21 +19,22 @@ We use the [milestone](https://github.com/rook/rook/milestones) feature in Githu
   * Longhaul testing pipeline [#1847](https://github.com/rook/rook/issues/1847)
   * Incorporate new environments [#1315](https://github.com/rook/rook/issues/1315)
```

Comment:
I remember a discussion that we would break apart some of these testing improvement items to beyond 1.0.  I think these can be moved to after 1.0:
Incorporate new environments #1315
Incorporate more architectures #1901